src/com/beginning/Transport/Main.java

```diff
@@ -0,0 +1,38 @@
+package com.beginning.Transport;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+public class Main implements Cloneable {
+    public static void main(String[] args) throws CloneNotSupportedException{
+        List <Car> firstCarShowroom = new ArrayList();
+        List <Plane> firstPlaneShowroom = new ArrayList();
+        // ArrayList<Car> secondCarShowroom = new ArrayList<>();
```

Comment:
First comment for you =) Uncommit this line just to check how the process looks like

Reply:
done